src/Sylius/Behat/Context/Api/Admin/ManagingShippingCategoriesContext.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Component\Shipping\Model\ShippingCategoryInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingShippingCategoriesContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    public function __construct(ApiClientInterface $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * @Given I want to create a new shipping category
+     */
+    public function iWantToCreateANewShippingCategory(): void
+    {
+        $this->client->buildCreateRequest('shipping_categories');
+    }
+
+    /**
+     * @Given /^I want to modify a (shipping category "([^"]+)")$/
```

Comment:
Could be simpler: `@Given I want to modify a shipping category :shippingCategory`